docs/user-guide/cai.rst

```diff
@@ -330,7 +327,7 @@ Run the following command to launch actor training and a reference policy server
 The trained LLM policy checkpoint will be saved to the output dir given by ``exp_manager.explicit_log_dir``.
 
 Step 7: Inference
```

Comment:
Line 332/329 revise step text so it is consistent with above and fix capitalization

Step 7: Run inference

Reply:
https://github.com/NVIDIA/NeMo-Aligner/pull/411/commits/9432333a44ad01d9534ed89263a37000b2aaca11